pysat/utils/io.py

```diff
@@ -1861,10 +1894,17 @@ def inst_to_netcdf(inst, fname, base_instrument=None, epoch_name=None,
 
                     # Use info in coltype to get real datatype of object
                     if coltype == str:
+                        if '_FillValue' in export_meta[lower_key].keys():
```

Comment:
Are strings allowed a fill_type in the netCDF standard?

Reply:
I don't see anything against it in the current guide: https://docs.unidata.ucar.edu/nug/current/best_practices.html

Reply:
Yeah, I don't see it either. Perhaps I'm thinking of something from ICON.

Anyway, while I reloaded this page several times of the past couple weeks I never saw this comment or other responses. This time I stared from a new browser window and voila.